tests/5.0/target_simd/test_target_simd_nontemporal.c

```diff
@@ -0,0 +1,52 @@
+//===--- test_target_simd_nontemporal.c -----------------------------------------------===//
+//
+// OpenMP API Version 5.0 Nov 2018
+//  
+// This test checks for support of the nontemporal clause on a target simd construct. 
+// The nontemporal clause indicates that accesses to the storage location of list items
+// have low temporal locality across the iterations in which those storage 
+// locations are accessed. 
+//  
+////===--------------------------------------------------------------------------------===/
+
+#include <omp.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include "ompvv.h"
+
+#define N 1028
+
+int test_simd_nontemporal() {
+   int errors = 0;
+   int i;
+   int a[N], b[N], c[N];
+
+   for (i = 0; i < N; i++) {
+      a[i] = 10; 
+      b[i] = i;
+      c[i] = 2 * i;
+   }   
+
+   #pragma simd target nontemporal (a, b, c)
+      for (i = 0; i < N; i += 100) {
```

Comment:
Should we turn `100` into a macro, called something like STRIDE_LEN? Just to allow some flexibility/avoid magic numbers.